javascript/javascript1/week1/week1-guzide/trialHouseyPriceyWfunction.js

```diff
@@ -0,0 +1,58 @@
+const customerArray = [
+  (peterHouse = {
```

Comment:
This causes an error in the console, please test your code before submitting. You can use https://stackblitz.com/ for quick tests. It allows you to spin up a development environment with different configurations - for now you can choose vanilla/static